src/main/java/net/imglib2/algorithm/morphology/table2d/Spur.java

```diff
@@ -73,7 +73,7 @@
 
 		final ImgFactory< T > factory;
 		if ( extendedVal instanceof NativeType )
-			factory = ( ImgFactory< T > ) Util.getArrayOrCellImgFactory( target, ( NativeType ) extendedVal );
+			factory = Util.< NativeType >getArrayOrCellImgFactory( target, ( NativeType ) extendedVal );
```

Comment:
I am using IntelliJ IDEA 2016.1.1, java 1.8.0_92 (previously 1.8.0_77), maven 3.3.9. Both compiling inside IDEA and running `mvn clean package` works. What's your java version?